AppDashboard/static/plugins/forms/tiny_mce/plugins/table/js/table.js

```diff
@@ -242,16 +242,16 @@ function insertTable() {
 	} else
 		inst.execCommand('mceInsertContent', false, html);
 
-	tinymce.each(dom.select('table[data-mce-new]'), function(node) {
+	tinymce.each(dom.select('table[data-mce-new]'), function(node) {

```

Comment:
Try to avoid committing changes like this unless you are reducing diffs with an external library.